Docs: Tablespaces note for primary-mirror pair

mperezfuster opened the pull request:
Tablespaces in different locations are not supported for a primary-mirror pair. This is different from filespaces so it has caused some confusion among customers.
I have edited the following 2 sections to add a warning about it:

https://mireia-tablespace.sc2-04-pcf1-apps.oc.vmware.com/7-0/admin_guide/ddl/ddl-tablespace.html
https://mireia-tablespace.sc2-04-pcf1-apps.oc.vmware.com/7-0/ref_guide/sql_commands/CREATE_TABLESPACE.html
https://mireia-tablespace.sc2-04-pcf1-apps.oc.vmware.com/7-0/install_guide/migrate.html

For the second link I didn't include it in a note element since there is a Notes section on that page. Let me know if this makes sense.

There is another pending PR to update the Release Notes for GPDB 6.0.
This PR should be merged to master and cherrypicked to 6.x.

ashwinstar reviewed (approved):
LGTM :+1: 

@soumyadeep2007 would you like us to add anything more on this topic in docs?

soumyadeep2007 commented:
> Greenplum Database does not support different tablespace locations for a primary-mirror pair with the same content ID. It is only possible to configure different locations for different content IDs. Symbolic links pointing to different file locations for a primary-mirror pair must not be used.

I think it might be better to **replace** **the** **last sentence** / **add:** "Modifying tablespace symlinks under pg_tblspc/ for a primary-mirror pair such that they point to different file locations will lead to erroneous behavior."

I want to include the above to drive home the point that one can only end up with different locations for a primary-mirror pair if they modify symlinks by hand **and it is an absolute no-no to do so**. My intent is to avoid the question: "if this is not supported why are they warning us?" Furthermore, since so many people have already done this, and many more will, it might be better to be explicit.

I would also prefer to see the light bulb in the second link for the same reason.

mperezfuster commented:
@soumyadeep2007 thanks for the suggestions, I have added them now. Also added a reference under https://mireia-tablespace.sc2-04-pcf1-apps.oc.vmware.com/6-16/install_guide/migrate.html as suggested by @Nirali13.

soumyadeep2007 reviewed (approved):
Awesome! This will help a lot of folks!

lisakowen commented on gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLESPACE.xml:
```diff
@@ -82,12 +82,17 @@
         command displays the tablespace OID values, names, and
         owner.<codeblock>SELECT oid, spcname, spcowner FROM pg_tablespace ;</codeblock></p>
-      <p>The Greenplum Database built-in function
+      <p>The Greenplum Databasse built-in function
```
Database?

mperezfuster replied:
woops

lisakowen commented on gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLESPACE.xml:
```diff
@@ -82,12 +82,17 @@
     </section>
+    <note>Greenplum Database does not support different tablespace locations for a primary-mirror
+      pair with the same content ID. It is only possible to configure different locations for
+      different content IDs. Modifying tablespace symbolic links under the directory pg_tblspc for a
```
codeph pg_tblspc?

lisakowen commented on gpdb-doc/dita/install_guide/migrate.xml:
```diff
@@ -93,6 +93,10 @@
             Resource Groups</xref> for information about enabling and configuring resource
           groups.</li>
+        <li>Filespaces are removed in Greenplum 6. When creating a tablespace, note that different
+          tablespace locations for a primary-mirror pair is now not supported. See <xref
```
what about ... now not -> no longer supported in Greenplum 6?

ielizaga reviewed (approved):
👍 - This is really helpful to have in our Documentation. Several users had tried to create symlinks to get this done, and that ended up causing problems with database utilities.
